components/Install/Install.tsx

```diff
@@ -1,19 +1,33 @@
-import { useState } from "react";
+import { useEffect, useState } from "react";
 
 import { Modal, Box, Fade, Backdrop, Typography } from "@mui/material";
 
 import { Collapse } from "antd";
+import { BeforeInstallPromptEvent } from "../../types";
 
-const Install = () => {
+interface IInstallProps {
```

Comment:
For typing please use types instead of interfaces. A correct use for interfaces is, for example, to define a data transfer object (DTO) like we use for Events, Shifts, Filters, Notifications, etc.

```suggestion
type IInstallProps = {
```